Sprintfinity3902/Content/Rooms/Room1.csv

```diff
@@ -11,3 +11,5 @@ WALT,-1,,,,,,,,,,
 WALB,-1,,,,,,,,,,
 WALL,-1,,,,,,,,,,
 ODRR,2,,,,,,,,,,
+2,7,,,,,,,,,,
+1
```

Comment:
These numbers and what the correlate to would be nice to have in the dictionary file.